wordpress-mobile/WordPress-Android
# Display author on post list items

Fixes #9576 

**Notes:** 
- We need to update FluxC's hash as soon as FluxC's PR is merged.
- https://github.com/wordpress-mobile/WordPress-Android/pull/10273 needs to be reviewed and merged first

Show author name on post list items.

To test:
Note: The author label will be shown only on posts which were fetched after the update.
1. My Site -> Blog posts
2. Select EVERYONE in the author filter
2. Notice the post list items contain author
3. Switch to list mode
4. Notice the post list items contain author
5. Select ME in the author filter and notice the author name is not shown

Update release notes:

- [ x ] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.

| <img width="489" alt="Screenshot 2019-07-24 at 11 57 14" src="https://user-images.githubusercontent.com/2261188/61784562-385c7480-ae0a-11e9-9f41-ac2a1ace3650.png">  |  <img width="488" alt="Screenshot 2019-07-24 at 11 55 15" src="https://user-images.githubusercontent.com/2261188/61784564-385c7480-ae0a-11e9-8282-fb22b8519fae.png"> |
|---|---|
